source/19-Psalms.usfm.db

```diff
@@ -7328,7 +7328,7 @@
 \q2
 \v 10 The fear of the \nd Lord\nd* is the beginning of wisdom
 \q those who keep it are wise indeed.
-\q2 His praise abides for ever and ever.
+\q2 His praise abides [us:forever|cth:for ever] and ever.
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".